src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -11,7 +11,6 @@ import {getMenu, isAppReady, dispatchSetMenu, dispatchOnAppReady} from '../../co
 import {dispatchHideDropDown, getLayouInfo, SHOW_DROPDOWN} from '../../core/LayoutCntlr.js';
 import {lcManager, LC} from './LcManager.js';
 import {LcResult} from './LcResult.jsx';
-import {LcPeriod} from './LcPeriod.jsx';
```

Comment:
LcPeriod.jsx was made on highchart, should this file be removed?

Reply:
yes, I will remove it

Reply:
@cwang2016 I don't see this at line 14 I think you were looking at the dev side of the diff.

Reply:
yes, the line is gone in this PR. Just asked to remove file LcPeriod.jsx.